src/cache.ts

```diff
@@ -142,10 +145,7 @@ export class LocalStorageCache implements ICache {
 
 export class InMemoryCache {
   public enclosedCache: ICache = (function () {
-    let cache: CachePayload = {
-      body: {},
-      expiresAt: 0
-    };
+    let cache: Record<string, CachePayload> = {};
```

Comment:
I am not sure I understand the intention of this change. This looks different from the original type ?

Reply:
Yeah I found this to be a bit weird, and the new type is more correct. For some reason `cache` was being initialised to an object but it was only ever interacted with using a key lookup. It should always have been a map rather than an object.

I don't believe the change breaks anything, it just means that the cache map doesn't have these extra `body` and `expiresAt` properties on them that shouldn't be there. This came up as an error as soon as turned on `noImplicitAny`, as some of the code around this was being implicitly typed as `any`.